instrumentation/instapgx/row.go

```diff
@@ -0,0 +1,39 @@
+// (c) Copyright IBM Corp. 2021
```

Comment:
New files no longer need the Instana header, and please fix the year to 2022 😵‍💫